Blazor.Bluetooth/wwwroot/JSInterop.js

```diff
@@ -333,6 +357,11 @@ window.ble.getDevices = async () => {
         throw 'Get devices is not supporting';
     } else {
         var devices = await navigator.bluetooth.getDevices();
+
+        devices.forEach((device) => {
+            PairedBluetoothDevices.push(device);
```

Comment:
```
    devices.forEach((device) => {
            var alreadyPariedDevice = getPairedBluetoothDeviceById(device.id);
            if (alreadyPariedDevice != null) {
                var indexToRemove = PairedBluetoothDevices.findIndex(x => x.id == device.id);
                PairedBluetoothDevices.splice(indexToRemove, 1);
            }

            PairedBluetoothDevices.push(device);
        });
```
We should remove the old device if it was already added to this collection.
Unfortunately can't test this code, because navigator.bluetooth.getDevices does not work for me, if it works for you please test it.
Due to the [documentation ](https://developer.mozilla.org/en-US/docs/Web/API/Bluetooth/getDevices) it won't be supported, but our's code should work.

Reply:
If you look at the documentation page you linked to it has a little white flag against the Chrome based browsers and Opera and if you click on the flag it details how to enable the getDevices functionality by setting the #enable-experimental-web-platform-features flag in chrome. 

I have tested it on Windows Chrome and Android Chrome and it works well. For my application we are only using devices that we control and so setting this flag is simple. For a public site this functionality not being released is pretty useless.

I cant understand why this feature has not been released yet as having to re-connect to the bluetooth device every time you reload the page severely limits the usability of Web Bluetooth.

Reply:
Thanks, I will complete your PR and will update a code for the library and test project. In a time when I started working on this, I was not able to run GetDevices and GetAvailability but now I tested and it works fine.
Thanks for your contribution I appreciate it.